docs/quick-start-guide.md

```diff
@@ -62,8 +62,9 @@ import org.apache.spark.sql.CarbonSession._
 * Create a CarbonSession :
 
 ```
-val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession()
+val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<hdfs store path>")
 ```
+NOTE: By default store location is pointed to "../carbon.metastore", user can provide own store location to CarbonSession like `SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<hdfs store path>", "<local metastore path>")`
```

Comment:
not store location, it is metastore location.  it can be as below
```
By default metastore(Derby DB) location is pointed to "../carbon.metastore", user can provide own metastore location to CarbonSession like 
```